.travis.yml

```
@@ -0,0 +1,29 @@
+language: python
+python:
+  - 3.8
+
+os:
+  - linux
+  - osx
+
+
+install:
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+
+  # Useful for debugging any issues with conda
+  - conda info -a
+
+  # Install Python, py.test, and required packages.
+  - pip install pytest
+  - conda env create -f environment-dev.yml
+  - source activate graphein-dev
+  - python setup.py install
+
+script:
+  # Your test script goes here
+  - pytest
```

```suggestion
  - pytest -vvv
```
Will give more information about the errors that occur, making it easier to fix the build.